Makefile

```diff
@@ -0,0 +1,4 @@
+branch = $(shell git branch | grep '*' | cut -d ' ' -f 2)
+
+sources:
```

Comment:
I'm not sure it's really needed since the specfile uses setup.py for these stages. I can add these targets invoking the setup.py, if you think it's worth it.